.ci/e2eKibana.groovy

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env groovy
+
+@Library('apm@current') _
+
+pipeline {
+  agent none
+  environment {
+    REPO = 'kibana'
+    BASE_DIR = "src/github.com/elastic/${env.REPO}"
+    GITHUB_CHECK_E2E_TESTS_NAME = 'E2E Tests'
+    PIPELINE_LOG_LEVEL = "INFO"
+  }
+  options {
+    timeout(time: 3, unit: 'HOURS')
+    buildDiscarder(logRotator(numToKeepStr: '20', artifactNumToKeepStr: '20', daysToKeepStr: '30'))
+    timestamps()
+    ansiColor('xterm')
+    disableResume()
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+    disableConcurrentBuilds()
+  }
+  // http://JENKINS_URL/generic-webhook-trigger/invoke
+  // Pull requests events: https://docs.github.com/en/developers/webhooks-and-events/github-event-types#pullrequestevent
+  triggers {
+    GenericTrigger(
+     genericVariables: [
+      [key: 'GT_REPO', value: '$.repository.full_name'],
+      [key: 'GT_BASE_REF', value: '$.pull_request.base.ref'],
+      [key: 'GT_PR', value: '$.issue.number'],
+      [key: 'GT_PR_HEAD_SHA', value: '$.pull_request.head.sha'],
+      [key: 'GT_BODY', value: '$.comment.body'],
+      [key: 'GT_COMMENT_ID', value: '$.comment.id']
+     ],
+    genericHeaderVariables: [
+     [key: 'x-github-event', regexpFilter: 'comment']
+    ],
+     causeString: 'Triggered on comment: $GT_BODY',
+     printContributedVariables: false,
+     printPostContent: false,
+     silentResponse: true,
+     regexpFilterText: '$GT_REPO$GT_BODY',
+     regexpFilterExpression: '^elastic/kibana/run-fleet-e2e-tests$'
```

Comment:
The command to run the tests in Kibana would be `run-fleet-e2e-tests`

Reply:
@EricDavisX this pipeline represents the way to interact with Kibana PRs. Will document it, but:

- In a kibana PR, add a comment with `/run-fleet-e2e-tests`. Only for elasticians
- Using the UI: trigger the pipeline using as the only parameter the GIT SHA of the kibana PR. (_We still have to capture the version of kibana for that commit SHA_)